.npmignore

```diff
@@ -1,7 +1,6 @@
 /*
 /dist/*
 !/dist/spectacle*
-/dist/*.map
```

Comment:
In the past we didn't ship `*.map` files because they were _huge_ and people complained about package size. What's our current source map size and do we risk running into those problems again?

Reply:
Our current map is pretty large (looking at around 700kB), I hadn't realised this was a problem and people had complained about package size. I'm happy to remove source maps as I suppose they're only at their most useful when developing Spectacle directly.